third_party/move/move-compiler-v2/tests/bytecode-generator/fv_enum_err.move

```diff
@@ -0,0 +1,32 @@
+module 0x66::fv_enum_basic {
```

Comment:
Could we also add the following test:
```
module 0xc0ffee::m {
    enum Wrapper {
        A(||u64 has copy),
        B(||u64 has copy + drop),
    }

    fun call(f: Wrapper): u64 {
        (f.0)()
    }

    public fun test(): u64 {
        let a = Wrapper::A(|| 42);
        call(a)
    }
}
```

We should get an error due to differing types on accessing `.0`.